bigquery/tests/test_export_data_to_cloud_storage.py

```diff
@@ -22,14 +22,32 @@
 
 class TestExportTableToGCS(CloudBaseTest):
 
-    def test_export_table(self):
+    def test_export_table_csv(self):
```

Comment:
These tests only check if the run() succeeds.
Is it possible to mock the client and check if the right RPCs are made?